nimare/generate.py

```diff
@@ -105,6 +111,61 @@ def create_coordinate_dataset(
     return ground_truth_foci, dataset
 
 
+def create_neurovault_dataset(
+    collection_ids=NEUROVAULT_IDS,
+    contrasts=CONTRAST_OF_INTEREST,
+    img_dir=None,
+    map_type_conversion=None,
+    **dset_kwargs,
+):
+    """
+    Parameters
```

Comment:
This needs a function description. Perhaps something like the following:

```suggestion
    """Download images from NeuroVault and use them to create a Dataset.

    Parameters
```